docs/source/models/causes/neonatal/index.rst

```diff
@@ -192,29 +238,26 @@ calculations.
       - Definition
       - Value or source
       - Note
-    * - mr
-      - neonatal mortality risk per live birth
-      - deaths_c380 / live_births
-      - The value of mr is a probabiity in [0,1]. Denominator includes
-        live births only.
-    * - deaths_c380
-      - cause-specific deaths due to neonatal disorders
-      - codcorrect
-      - just for neonatal age groups
-    * - live_births
-      - birth rate (live births only)
-      - sum(ASFR*population)
-      - Units are total live births  per person-year
-    * - ASFR
-      - Age-specific fertility rate
-      - get_covariate_estimates: coviarate_id=13
-      - Assume lognormal distribution of uncertainty. Units in GBD are
-        live births per person, or equivalently, per person-year.
-    * - population
-      - average population in a given year
-      - get_population
-      - Specific to age/sex/location/year demographic group. Numerically
-        equal to person-time for the year.
+    * - ACMR
+      - all-cause mortality rate (per person year)
+      - GBD
```

Comment:
Usually we're more specific about how to get the data from. E.g., to get ACMR, you probably have to pull deaths (with source = 'codcorrect'?) using `get_draws`, and pull population using `get_population`, then divide them. Perhaps the engineers have done this enough by now that we don't need to be so explicit anymore, but that's traditionally how we lay out these tables.

Reply:
Can I use vivarium_inputs and gbd_mapping calls?  I don't want to look up all of the magic numbers for get_draws.

Reply:
I guess we can leave it as is for now and see if the engineers complain.